src/index.js

```diff
@@ -180,6 +181,20 @@ class Client<RequestType, ResponseType> {
     return this.listResource(`projects/${projectID}/stories`);
   }
 
+  /** */
+  searchStories(
+    query: String,
+    pageSize: number = 25,
+  ): Promise<StorySearchResult> {
+    return this.getResource(`search/stories`, {
+      query,
+      page_size: pageSize,
+    }).then(result => ({
+      data: result.data,
```

Comment:
What do you think about spreading `result` here, instead of plucking out the `data` field? That way, if other new fields are returned here, this implementation won't need to change too.
```suggestion
      ...result
```

Reply:
This will also allow consumers to get to the `next` field, in case they want to use it in a way other than how `fetchNext` works.

Reply:
I update the PR with a spread.